packages/admin/admin/src/mui/menu/Menu.tsx

```diff
@@ -79,17 +83,13 @@ const MenuDrawer: React.FC<WithStyles<typeof styles> & MenuProps> = ({
 
             <Drawer
                 variant="permanent"
+                drawerWidth={drawerWidth}
+                drawerWidthCollapsed={drawerWidthCollapsed}
                 className={permanentDrawerClasses.join(" ")}
                 open={permanentOpen}
-                style={{ width: permanentOpen ? drawerWidth : 0 }}
+                hidden={variant === "temporary"}
                 PaperProps={{
                     elevation: 2,
-                    style: {
-                        top: headerHeight,
-                        height: `calc(100% - ${headerHeight}px)`,
-                        width: drawerWidth,
-                        marginLeft: permanentOpen ? 0 : -drawerWidth,
-                    },
```

Comment:
Is it really okay to remove this?

Reply:
Jup because we need no top offset in the new design on mobile.